app/static/js/createEvents.js

```diff
@@ -35,6 +35,11 @@ function format24to12HourTime(timeStr){
  * Run when the webpage is ready for javascript
  */
 $(document).ready(function() {
+  $("#saveEventBtn").click(function() {
+      $("#saveEvent").submit()
```

Comment:
Doesn't this double submit? When you click a submit button the form will submit automatically. Your onclick should just disable the button

Reply:
No, disabling the button somehow makes the form not submit.